### ISSUE TITLE
api load: filestore: Error getting migrated host: unexpected end of JSON input

### ISSUE THREAD

**Issue by rameshpuruganti** [kubernetes/minikube#5046]
**The exact command to reproduce the issue**:
 minikube start --vm-driver hyperv --hyperv-virtual-switch="Minikube" 
This is the command i am running. 

**The full output of the command that failed**:<details>
The output is 
minikube v1.3.0 on Microsoft Windows 10 Pro 10.0.17763 Build 17763
E0812 11:39:52.134815   15164 start.go:723] StartHost: Error loading existing host. Please try running [minikube delete], then run [minikube start] again.: filestore: Error getting migrated host: unexpected end of JSON input

X Unable to start VM: Error loading existing host. Please try running [minikube delete], then run [minikube start] again.: filestore: Error getting migrated host: unexpected end of JSON input

</details>

**The output of the `minikube logs` command**:<details>

In Log it is printed as: 
X api load: filestore: Error getting migrated host: unexpected end of JSON input

</details>

**The operating system version**: Windows 10 Pro

**Comment by athurner** [comment]
I had a similar issue after upgrading from 1.2. Deleting the c:\Users\<username>\.minikube folder solved my issue.

**Comment by tstromberg** [comment]
Interesting. Could you attach the `config.json` file? it should be in. `c:\Users<username>.minikube\machines\minikube`

The instructions should work: running `minikube delete` should resolve the issue.

**Comment by rameshpuruganti** [comment]
config.json file is empty. 
minikube delete is not working. i am getting same error. Is there any way to delete the folders from the windows and reinstall it again? I have tried to delete the folders manually some are deleted and some are not deleting since it is saying the folders are being used by some other services. Please guide me for the clean and full installation

**Comment by tstromberg** [comment]
Confirmed on macOS:

```
 ~/.m/m/minikube  cat > config.json
 ~/.m/m/minikube  minikube delete

💣  Failed to delete cluster: Error getting migrated host: unexpected end of JSON input

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new/choose
```

**Comment by tstromberg** [comment]
Thank you for the update! 

As far as deleting folders in Windows, I'm not sure how to guide you to doing so with full permissions. You may need to stop the VM in Hyper-V first if one exists.

**Comment by rameshpuruganti** [comment]
Thank you for your response. I have tried that as well by stopping Hyper-V and delete the folders. But somehow its not deleting and throwing message like the file or folder is being used.
Not sure which service is using this file. I have stopped all the services related to minikube. STILL THE ISSUE EXISTS :(

**Comment by tstromberg** [comment]
@rameshpuruganti I believe this issue is now addressed by minikube v1.4, as it forcibly resets VM's that are in a failed state. If you still see this issue with minikube v1.4 or higher, please reopen this issue by commenting with `/reopen`

Thank you for reporting this issue!
